src/app/views/public/landing/landing.html

```diff
@@ -0,0 +1,82 @@
+
+<div id="public">
+  <nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container-fluid">
+      <div class="navbar-header pull-left">
+        <div class="nav-logo pull-left"></div>
+        <div class="navbar-brand">
+          {{::app_name}}
+        </div>
+      </div>
+      <div class="links pull-right">
+        <a href="/mnoe/auth/users/sign_up" class="btn btn-default pull-right">Sign-up</a>
+        <a href="/mnoe/auth/users/sign_in" class="btn btn-default pull-right">Login</a>
```

Comment:
I think you can add/reuse the links constant in `/mno-enterprise-angular/src/app/index.constants.coffee`